### ISSUE TITLE
"Naked and Afraid" profession should potentially give more points

### ISSUE THREAD

**Issue by NetSysFire** [CleverRaven/Cataclysm-DDA#61728]
### Describe the bug

A hobo with both more equipment and even food to start with gives you one point. Even a shower victim gives you the same amount of points.

### Steps to reproduce

n/a

### Expected behavior

A profession which literally does not start with anything should give you more points over one that has at least some basics (such as a knife or even some clothing).

However, we can not use floating point numbers, so e.g giving 1.5 points is not possible.
Not really game breaking, but this has been annoying me for a while.

### Versions and configuration

n/a

**Comment by robotic-gamer** [comment]
one possible solution would be to add a useful trait to the start. one option would be some positive but rarely used trait like outdoorsman. or a new trait like nudism which gives a mood bonus dependent on how much of your character is not covered by clothing.

**Comment by ZhilkinSerg** [comment]
Isn't it from challenge scenarios?

**Comment by Stone94** [comment]
> Isn't it from challenge scenarios?

No, but I don't see any issue with naked and afraid, its challenge only lasts several minutes outside of a winter or wilderness start (at least for me). I'd lean more to making hobo cost 0 since it starts with clothes, a storage option, a pocket knife, matches and a can of food. This gives immediate access to boiling water and food but that's just my opinion, the main reason hobo gave points initially was because of the alcohol addiction and frequency that you would catch a cold at the time.

**Comment by TheShadowFerret** [comment]
#58689 will give you 8 points currently if you give up all your items, if it were to be merged(but it's feature frozen) 🤔
